metabrainz/templates/index/contact.html

```diff
@@ -52,36 +52,33 @@ <h1 class="page-title">Contact</h1>
       disallowed topics, we reserve the right to make fun of you.
     </em>
   </p>
+
+  <h3>E-mail us</h3>
   <button id="agree-btn" class="btn btn-default btn-success">
-    I agree that I will not contact you regarding the topics above
+    I agree with rules above, show me email address
   </button>
-
-  <div id="contact-details" style="display: none;">
-
-    <h3>E-mail us</h3>
+  <div id="email-address" style="display: none;">
     <p>You can reach the MetaBrainz staff directly at the following address:</p>
-    <p>
-      <span style="font-size: 110%">
-        <strong><a href="mailto:support@metabrainz.org">support@metabrainz.org</a></strong>
-      </span>
+    <p style="font-size: 110%">
+      <strong><a href="mailto:support@metabrainz.org">support@metabrainz.org</a></strong>
     </p>
+  </div>
 
-    <h3>IRC</h3>
-    <p>
-      The MetaBrainz/MusicBrainz team, as well as MusicBrainz users, hang out in the
-      <strong><a href="irc://irc.freenode.net/metabrainz">#metabrainz</a></strong>
-      channel on IRC chat at irc.freenode.net. Please feel free to browse the
-      <a href="http://chatlogs.metabrainz.org/">IRC log archives</a>.
-    </p>
+  <h3>IRC</h3>
+  <p>
+    The MetaBrainz/MusicBrainz team, as well as MusicBrainz users, hang out in the
+    <strong><a href="irc://irc.freenode.net/metabrainz">#metabrainz</a></strong>
+    channel on IRC chat at irc.freenode.net. Please feel free to browse the
+    <a href="http://chatlogs.metabrainz.org/">IRC log archives</a>.
+  </p>
 
-    <h3>Mailing address</h3>
-    <p>
-      MetaBrainz Foundation<br />
-      3565 South Higuera St, Suite B<br />
-      San Luis Obispo, CA 93401<br />
-      United States
-    </p>
-  </div>
+  <h3>Mailing address</h3>
```

Comment:
Unsure whether this bit should be hidden or shown. @mayhem? I guess we don't really read it anyway...